web3.md

```diff
@@ -741,6 +750,35 @@ eth_sendRawTransaction
     rule <k> #eth_sendRawTransactionSend TXID => #sendResponse( "result": "0x" +String #hashSignedTx( TXID ) ) ... </k>
 ```
 
+Transaction Receipts
+--------------------
+- The transaction receipt is a tuple of four items comprising: 
+  - the cumulative gas used in the block containing the transaction receipt as of immediately after the transaction has happened
+  - the set of logs created through execution of the transaction
+  - the Bloom filter composed from information in those logs
+  - the status code of the transaction.
+
+```k
+    syntax KItem ::= "#makeTxReceipt" Int
+ // -------------------------------------
+    rule <k> #makeTxReceipt TXID => . ... </k>
+         <txReceipts>
+           ( .Bag
+          => <txReceipt>
+               <txHash> "0x" +String #hashSignedTx (TXID) </txHash>
+               <txCumulativeGas> CGAS </txCumulativeGas>
+               <logSet> LOGS </logSet>
+               <bloomFilter> #bloomFilter(LOGS) </bloomFilter>
+               <txStatus> bool2Word(STATUSCODE ==K EVMC_SUCCESS) </txStatus>
+             </txReceipt>
```

Comment:
It looks like here https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionreceipt that we'll also need the gas used by the transaction itself, _not_ just the cumulative gas used.

You can calculate it by doing `<txGasLimit> -Int <gas>`, I believe. Store it in a new cell `<txGas>`.

Reply:
The tx receipt described in JSON RPC also contains some block info and has a few extra fields like `txGas`. I've only implemented the ones which are documented in the yellowpaper.
https://ethereum.github.io/yellowpaper/paper.pdf 
```
4.3.1:
...
The transaction receipt,R, is a tuple of four items com-
prising:  the cumulative gas used in the block containing
the transaction receipt as of immediately after the trans-
action has happened,Ru, the set of logs created through
execution of the transaction,Rland the Bloom filter com-
posed from information in those logs,Rband the status
code of the transaction,
Rz:(20)R≡(Ru,Rb,Rl,Rz)
```